test/test_parse_es.py

```diff
@@ -84,6 +86,134 @@ def test_extract_number_es(self):
             lang='es')), [1, 4, 7, 8, 14, 157])
         self.assertEqual(extract_number("seis punto dos", lang='es'), 6.2)
         self.assertEqual(extract_numbers("un medio", lang='es'), [0.5])
+        self.assertEqual(extract_number("cuarto", lang='es'), 0.25)
+
+        self.assertEqual(extract_number("2.0", lang='es'), 2.0)
+        self.assertEqual(extract_number("1/4", lang='es'), 0.25)
+
+        self.assertEqual(extract_number("dos e media", lang='es'), 2.5)
+        self.assertEqual(extract_number(
+            "catorce e milésima", lang='es'), 14.001)
```

Comment:
In spanish we would say: "catorce coma cero cero uno" I hardly doubt someone says "catorce y una milésima"

Reply:
Agreed. This test is just to make sure that it 1) catches milésima, and 2) correctly combines it with another number. You'd hardly ever say "14 and a thousandth" in English, either, but edge cases are half the point.

That's also what these 'e's were for - cover edge cases - but they're obviously superfluous now that 'e' has been removed. I'd never noticed there aren't any numbers ending in 'i'. I just dutifully made sure both ands were functional.